### PR TITLE
Update html-cheatsheet.md

### PR DESCRIPTION
**Pull request by ritwikdalmia**

### CODE REVIEW

File: CheatSheets/html-cheatsheet.md [hunk 1]
```diff
@@ -150,11 +153,88 @@
...
+|:-- |:-- |:--  |
+| plain text | `<input type="text">` | the type attribute can be omitted |
+| password field | `<input type="password">`  | echoes dots instead of characters |
+| text area | `<textarea></textarea>` |  a more customizable plain text area |
+| checkbox | `<input type="checkbox">` | can be toggled on or off |
+| radio button | `<input type="radio">`  can be grouped with other inputs |
+| drop-down lists | `<select><option>` | [check here for more info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) |
+| file picker | `<input type="file">`  | pops up an “open file” dialog |
+| hidden field | `<input type="hidden">`   | nothing there!
+| submit button | `<input type="submit">`  | activates the form's submission <br/>(a `POST` request or <br/>Javascript action) |
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:212]: Table format is broken [here](https://github.com/ritwikdalmia/CheatSheets-for-Developers/blob/ritwikdalmia-html-cheetsheet/CheatSheets/html-cheatsheet.md#input-types)

File: CheatSheets/html-cheatsheet.md [hunk 2]
```diff
@@ -150,11 +153,88 @@
...
+<input id="user" type="text" name="name" />
+```
+
+#### `<label>`'s Attributes
+
+**Very important!** `for=` in a label references an `<input>`s `id=` attribute, not it's `name=` attribute!
+Sometimes these values will be the same, but `for=` goes with `id=`. `name=` is the key of the `<input>`'s 
+value when it arrives at the server.
+
+**[🔼Back to Top](#table-of-contents)**
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:197]:
```suggestion
```
Please, follow the tabular format. You didn't follow the [contributing guidelines](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/docs/CONTRIBUTING.md).
This is a CheatSheet, not an elaborate tutorial. Use `one row` in a table for a `single tag`.
| Command | Description |
| -------- | ---------------- |
| `<label> ... </label>` | Define a label for an HTML form widget |

File: CheatSheets/html-cheatsheet.md [hunk 3]
```diff
@@ -9,6 +9,9 @@
   - [Images](#images)
   - [Lists](#lists)
   - [Forms](#forms)
+  - [Important Attributes](#important-attributes)
+  - [INPUT Types](#input-Types)
+  - [The `<label>` Element (Tag)](#The-label-Element)
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:14]: Don't create unnecessary Headings.

File: CheatSheets/html-cheatsheet.md [hunk 4]
```diff
@@ -150,11 +153,88 @@
...
+#### `<label>`'s Attributes
+
+**Very important!** `for=` in a label references an `<input>`s `id=` attribute, not it's `name=` attribute!
+Sometimes these values will be the same, but `for=` goes with `id=`. `name=` is the key of the `<input>`'s 
+value when it arrives at the server.
+
+**[🔼Back to Top](#table-of-contents)**
+
+
+## INPUT Types
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:200]:
```suggestion
```
This section will be quite useful if you keep it simple, meaningful, and readable. Please, maintain the `tabular format`.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify your codes.

**Comment by ritwikdalmia** [comment]
i have resolve all the conflict please check once again

**Comment by ritwikdalmia** [comment]
please merge this PR

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**